src/lib/utils/SynapseClient.tsx

```diff
@@ -516,6 +558,18 @@ export const getSessionTokenFromCookie = () => {
   const sessionToken = localStorage.getItem(DEV_ENV_SESSION_LOCAL_STORAGE_KEY)
   return Promise.resolve(sessionToken)
 }
+/**
+ * Get the current session token from a cookie.  Note that this will only succeed if your app is running on
+ * a .synapse.org subdomain.
+ */
+export const refreshSessionToken = () => {
```

Comment:
I'm confused, this looks like a copy of getSessionTokenFromCookie (above it)

Reply:
whoops, that was an accident